src/include/migraphx/op/squeeze.hpp

```diff
@@ -55,7 +55,15 @@ struct squeeze
                 }
             }
         }
-        return shape{type, new_lens};
+
+        if(new_lens.empty())
+        {
+            return shape{type};
+        }
+        else
+        {
+            return shape{type, new_lens};
+        }
```

Comment:
@kahmed10 I am not sure how this affects the squeeze change for TF.

Reply:
I will test once this change gets merged in.